packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/sagas.ts

```diff
@@ -7,14 +8,30 @@ import { selectors } from 'data'
 import { actions as A } from './slice'
 
 export default ({ api }: { api: APIType }) => {
+  const getCards = function* () {
+    try {
+      const data = yield call(api.getDCCreated)
```

Comment:
why no use of a Loading remote state?  how are you able to render a loading indicator to user?

Reply:
This is requested at the beginning of the wallet page, no loading was defined to be shown anywhere